src/Compute/Compute/Generated/DedicatedHostGroup/DedicatedHostGroupGetMethod.cs

```diff
@@ -56,21 +56,18 @@ public override void ExecuteCmdlet()
 
                 if (!string.IsNullOrEmpty(resourceGroupName) && !string.IsNullOrEmpty(hostGroupName))
                 {
+                    DedicatedHostGroup result;
                     if (this.InstanceView.IsPresent)
                     {
-                        var result = DedicatedHostGroupsClient.Get(resourceGroupName, hostGroupName, InstanceViewTypes.InstanceView).InstanceView;
-                        var psObject = new PSHostGroupInstanceView();
-                        ComputeAutoMapperProfile.Mapper.Map<DedicatedHostGroupInstanceView, PSHostGroupInstanceView>(result, psObject);
-                        WriteObject(psObject);
+                        result = DedicatedHostGroupsClient.Get(resourceGroupName, hostGroupName, InstanceViewTypes.InstanceView);
                     }
                     else 
                     {
-                        var result = DedicatedHostGroupsClient.Get(resourceGroupName, hostGroupName);
-                        var psObject = new PSHostGroup();
-                        ComputeAutomationAutoMapperProfile.Mapper.Map<DedicatedHostGroup, PSHostGroup>(result, psObject);
-                        WriteObject(psObject);
+                        result = DedicatedHostGroupsClient.Get(resourceGroupName, hostGroupName);
                     }
-
+                    var psObject = new PSHostGroup();
+                    ComputeAutomationAutoMapperProfile.Mapper.Map<DedicatedHostGroup, PSHostGroup>(result, psObject);
+                    WriteObject(psObject);
                 }
                 else if (!string.IsNullOrEmpty(resourceGroupName))
                 {
```

Comment:
if InstanceView is present, shouldn't 'result' be mapped to PSHostGroupIntanceView?

Reply:
interesting, maybe thats why the test was failing yesterday. @madewithsmiles

Reply:
@madewithsmiles